tools/cell_census_builder/util.py

```diff
@@ -112,3 +114,10 @@ def anndata_ordered_bool_issue_853_workaround(df: pd.DataFrame) -> pd.DataFrame:
             df[k] = df[k].cat.set_categories(df[k].cat.categories, ordered=bool(df[k].cat.ordered))
 
     return df
+
+def get_git_commit_sha() -> str:
+    """
+    Returns the git commit SHA for the current repo
+    """
+    repo = git.Repo(search_parent_directories=True)
+    return repo.head.object.hexsha
```

Comment:
If we want to be super paranoid (and who doesn't want to be that!?), we could also ensure the git workspace is clean (no uncommitted changes).  Not critical, but might help while we're still building the census manually. Discourages build-time "hot fixes" that aren't committed.

Reply:
Good idea. Added.